fiaas_skipper/web/static/skipper.js

```diff
@@ -17,22 +17,28 @@ function getStatus() {
                        + "<td>" + item.status + "</td>"
                        + "<td>" + [item.description].join('') + "</td>"
                        + "<td>" + item.version + "</td>"
-                       + "<td><button class=\"btn btn-primary btn-block btnDeploy\" type=\"submit\" data-namespace=\"" + item.namespace + "\">Deploy</button></td>"
+                       + "<td><button class=\"btn btn-primary btn-block btnDeploy\" type=\"submit\" data-namespace=\"" + item.namespace + "\" data-toggle=\"modal\" data-target=\"#deployModal\">Deploy</button></td>"
                        + "</tr>";
            $('#tbody').append(eachrow);
        });
-       $(".btnDeploy").each(function(){
-           var $this = $(this);
-           $this.click(function(){
-               $.ajax({
-                   type: "POST",
-                   contentType: "application/json",
-                   url: "/api/deploy",
-                   data: "{\"namespaces\": [\"" + $(this).data('namespace') + "\"]}",
-                   success: function(data) {
-                       alert("Deployment to " + $this.data("namespace") + " scheduled successfully");
-                   }
-               });
+       $("#deployModal").on("show.bs.modal", function (event) {
+            var button = $(event.relatedTarget);
+            var namespace = button.data('namespace');
+            var modal = $(this);
+            modal.find(".modal-title").text('Deploy fiaas-deploy-daemon to namespace: " + namespace + "?");
```

Comment:
Wrong type of quote at the beginning of the string here?

Reply:
Yeah. Fixed.